web-app/CKS.Web.Test/CKS.Web.Test.csproj

```diff
@@ -22,9 +22,36 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="appsettings.test.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Should these all be the same `CopyToOutputDirectory`? Some are Always and some are PreserveNewest